scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/ValidationUtil.java

```diff
@@ -98,6 +94,12 @@ public static void validate(File jobFile, InputStream schemaIs)
         }
     }
 
+    public static byte[] getInputStreamBytes(InputStream inputStream) throws IOException {
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        IOUtils.copy(inputStream, outputStream);
+        return outputStream.toByteArray();
```

Comment:
I am not sure but maybe it is better to save result, and close outputStream. i am not sure

Reply:
I also added a try-with-resource there.